source/Symbol/SwiftASTContext.cpp

```diff
@@ -1276,74 +1284,93 @@ static void ConfigureResourceDirs(swift::CompilerInvocation &invocation,
   invocation.setRuntimeResourcePath(resource_dir.GetPath().c_str());
 }
 
-/// Initialize the compiler invocation with it the search paths from a
-/// serialized AST.
-/// \returns true on success.
-static bool DeserializeCompilerFlags(swift::CompilerInvocation &invocation,
-                                     StringRef section_data_ref, StringRef name,
-                                     llvm::raw_ostream &error) {
-  auto result = invocation.loadFromSerializedAST(section_data_ref);
-  if (result == swift::serialization::Status::Valid)
-    return true;
 
-  error << "While deserializing" << name << ":\n";
-  switch (result) {
+static const char *getImportFailureString(swift::serialization::Status status) {
+  switch (status) {
   case swift::serialization::Status::Valid:
-    llvm_unreachable("already checked");
+    return "The module is valid.";
   case swift::serialization::Status::FormatTooOld:
-    error << "The swift module file format is too old to be used by the "
-             "version of the swift compiler in LLDB\n";
-    break;
-
+    return "The module file format is too old to be used by this version of "
+           "the debugger.";
   case swift::serialization::Status::FormatTooNew:
-    error << "the swift module file format is too new to be used by this "
-             "version of the swift compiler in LLDB\n";
-    break;
-
+    return "The module file format is too new to be used by this version of "
+           "the debugger.";
   case swift::serialization::Status::MissingDependency:
-    error << "the swift module file depends on another module that can't be "
-             "loaded\n";
-    break;
-
-  case swift::serialization::Status::MissingUnderlyingModule:
```

Comment:
This is the new name for `MissingShadowedModule` https://github.com/apple/swift/commit/6cca050fd9